docs/architecture/adr-013-metrics.md

```diff
@@ -0,0 +1,97 @@
+# ADR 013: Observability
+
+## Changelog
+
+- 20-01-2020: Initial Draft
+
+## Status
+
+Proposed
+
+## Context
+
+### A
+
+There has been discussion around exposing more metrics to users and node operators about the application. Currently there is only a way to expose metrics from Tendermint and not the application itself.To bring more visibility into applications, I would like to propose reporting of metrics through [Prometheus](https://prometheus.io/).
+
+#### List Of Metrics
+
+(TBD)
+
+| Name                  |   type    | Description |
+| --------------------- | :-------: | ----------: |
+| iavl_io               |  Gauage   |             |
```

Comment:
Gauge, I think we do need description and also what units are we measuring io as well.